guest/testdata/cyclestate/main.go

```diff
@@ -153,5 +170,7 @@ func mustFilterState(state api.CycleState) {
 		panic("didn't propagate state from pre-filter")
 	} else if _, ok = val.(preFilterStateVal)["filter"]; !ok {
 		panic("filter value lost propagating from pre-score")
+	} else if _, ok = val.(preFilterStateVal)["postfilter"]; !ok {
+		panic("postfilter value lost propagating from pre-score")
```

Comment:
I don't think we need this change. As mentioned earlier, no extension point is called after postfilter.

Reply:
Fixed it, thanks.